### PR TITLE
Vet clinic database: query and update animals table 

### PR DESCRIPTION
**Pull request by berylcl**
IN THIS PR have Included,
Add a column species of type string to your animals table. Modify your schema.sql file.
Insert the following data:
  - Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
  - Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
  - Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
  - Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
  - Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
  - Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
  - Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.
    Remember to add these insertions to your data.sql file.
Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
Inside a transaction:
    - Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
    - Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
    - Verify that changes were made.
    - Commit the transaction.
    -  Verify that changes persist after commit.
Screenshot from 2023-07-13 10-01-11
     - Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
     - After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)
    Screenshot from 2023-07-13 10-03-55
Inside a transaction:
  - Delete all animals born after Jan 1st, 2022.
  - Create a savepoint for the transaction.
  -  Update all animals' weight to be their weight multiplied by -1.
  -  Rollback to the savepoint
  - Update all animals' weights that are negative to be their weight multiplied by -1.
  - Commit transaction
  Screenshot from 2023-07-13 10-06-53
  - Write queries to answer the following questions:
    - How many animals are there?  
Screenshot from 2023-07-13 10-08-14
    - How many animals have never tried to escape?
Screenshot from 2023-07-13 10-09-13
    - What is the average weight of animals?
Screenshot from 2023-07-13 10-09-52
    - Who escapes the most, neutered or not neutered animals?
Screenshot from 2023-07-13 10-10-31
    - What is the minimum and maximum weight of each type of animal?
Screenshot from 2023-07-13 10-11-03
    - What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Screenshot from 2023-07-13 10-11-36

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -22,4 +22,85 @@ SELECT * FROM animals WHERE neutered = true;
...
+-- Rollback to the savepoint
+ROLLBACK TO my_savepoint;
+
+-- Update all animals' weights that are negative to be their weight multiplied by -1
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+-- Commit the transaction
+COMMIT;
+-- Begin the transaction
+BEGIN;
```
Comment (DeliceLydia) [inline queries.sql]: - [ ] This query was repeated instead this was about ` deleting all records in the animal's table, then rolling back the transaction.`
```sql
BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```

Comment (DeliceLydia) [reply queries.sql]: - [ ] Please `UPDATE` the animals table by setting the `species` column to unspecified, then roll back the change
```sql
 BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; 
```

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @berylcl, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Followed git workflow
- Good README


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @berylcl 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**